03-task-manager/starter/quizAnswers2.txt

```diff
@@ -0,0 +1,13 @@
+1. In this lesson, you created a middleware function called asyncWrapper. Why?
+    asyncWrapper wrapper is used to handle async errors thrown by the controllers.
+
+2. Suppose that you want to make sure that both a status code and and error message are send back to the user when they request the URL for a task that does not exist. Assume that you’ve created a CustomAPIError class and an error handler that references that class. Complete the code:
+const getTask = asyncWrapper(async (req, res, next) => {
+  const { id: taskID } = req.params;
+  const task = await Task.findOne({ _id: taskID });
+  iSf (!task) {
```

Comment:
```suggestion
  if (!task) {
```

Reply:
@hammaazarok did you get a chance to fix this? Then you'll be able to merge this PR when you're ready

Reply:
Just fixed,  thank you for the review.